src/main/java/org/jenkinsci/pipeline_steps_doc_generator/PipelineStepExtractor.java

```diff
@@ -206,4 +224,106 @@ public void generateAscii(Map<String, Map<String, List<StepDescriptor>>> allStep
             }
         }
     }
+
+    public void generateDeclarativeAscii() {
```

Comment:
Think this could be pulled out of this class?  I hoped that `PipelineStepExtractor` would act more as a hub that goes about calling other classes to do the heavy lifting.

Reply:
Fair enough. =)